public/app.js

```diff
@@ -19,7 +19,9 @@ function updateDom (err, data) {
     var users = JSON.parse(data);
     var table = document.getElementById("users-table");
     /* create a row in table for each user returned from DB */
+    console.log('hello', users);
```

Comment:
Hmm I thought they were from the original but looks like I might have added them. Removing them now.